### PR TITLE
chore: add new accordion and step components

### PR DESCRIPTION
**Pull request by cjbell**
### Description

- Add new `AccordionGroup` and `Accordion`
- Add new `Steps` and `Step`
- Add new `/playground` page

### Screenshots

image

### CODE REVIEW

File: content/playground.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+> This is a block quote
+
+- An
+- Unordered
+- List
+
+1. A
```
Comment (cellomatt) [inline content/playground.mdx:25]: Not sure whether we care about grammar on the playground, but if so:
```suggestion
1. An
```

File: components/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    role="list"
+  >
+    {children}
+  </div>
+);
+
+type AccordionProps = {
+  title: string;
+  description?: string;
+  defaultOpen?: boolean;
```
Comment (cellomatt) [inline components/Accordion.tsx:17]: nice touch 👌

**Review by cellomatt — APPROVED** [review]
🔥

**Review by JEverhart383 — APPROVED** [review]
100

**Review by connorlindsey — APPROVED** [review]
Nice! Really clean components

**Pull request merged**